Release notes for 0.111.0 (#13654)
home-assistant/home-assistant.github.io: source/_posts/2020-06-10-release-111.markdown

---
layout: post
title: "0.111: Beta release notes"
description: "0.111: Beta release notes"
date: 2020-06-01 00:00:00
date_formatted: "June 10, 2020"
author: Franck Nijhof
author_twitter: frenck
comments: true
categories: Release-Notes
og_image: /images/blog/2020-06-0.111/social.png
---

<a href='/integrations/#version/0.111'><img src='/images/blog/2020-06-0.111/components.png' style='border: 0;box-shadow: none;'></a>

Home Assistant Core 0.111 is here!

So, lets face it: the previous release (0.110) was just jam packed with
new features, tons of upgrades and a lot of stuff changing. It was pretty
exciting! It will be hard to top that.

Personally, I'm always looking forward to the new features a new release brings.
Time to play! This time, however, not so much.

This release is focussed around more stability, fixing, tweaking and tuning.

> Honestly, I think it it really nice!

Reviewer: ```suggestion
Honestly, I think it is really nice!
```

Most notably, is the change on how Home Assistant loads up the frontend.
It is available really quick now!

Enjoy the release!

../Frenck

## Starting the frontend sooner

In this version, we start the Home Assistant frontend and API server before all
integrations are loaded. This means you can interact with Home Assistant
sooner than before.

> Instead of waiting a couple of minutes before the Home Assistant frontend

Reviewer: ```suggestion
Instead of waiting a couple of minutes until the Home Assistant frontend
```

becomes responsive, it is available really fast now!

However, with this change, Home Assistant no longer waits for all integrations

> to be ready. A result, not all devices and entities are available immediately.

Reviewer: ```suggestion
to be ready. As a result, not all devices and entities are available immediately.
```

This is actually good! As this means, an integration that got into trouble,
can no longer prevent the frontend from becoming available. Also, as soon

> as it is available, you can change or remove the configuration of an non-working

Reviewer: ```suggestion
as it is available, you can change or remove the configuration of a non-working
```

integration. Finally, it easier to check out your logs when something
goes wrong.

The base for this change came from [@bdraco] his creative brains, so thanks
for that! [@bramkragten] did all the frontend work and [@pvizeli] made sure
the Supervisor handles the surprisingly fast available frontend as well.

Great work guys!

One additional note: If you run generated Lovelace, it will still wait for
Home Assistant to be completely started. If you created your own dashboard,
it will show warnings for entities that are not available yet and will update
when they become available.

## Other startup improvements

Some more tuning to the startup process can be found in things like the logs.

> If an integration takes more time to set up, it will be shown in the logs.

Reviewer: ```suggestion
If an integration takes more time to set up, it will be shown in the logs every 60 seconds, indicating that the integration is still being setup.
```

While is is loading, every 60 seconds a log message will be shows that we are

> awaiting the integration to be set up.

Reviewer: replace with above

> Another speed improvement is found in the way we load up integration itself.

Reviewer: ```suggestion
Another speed improvement is found in the way we load up integrations themselves.
```

> Often integration have a basic setup and will load the various platforms

Reviewer: ```suggestion
Often, an integration has a basic setup and will then load the various platforms
```

(like lights and switches) after that.

> As of this release, Home Assistant will set up the integration and but will

Reviewer: ```suggestion
As of this release, Home Assistant will set up the integration but will
```

> no longer wait for the platforms finishing setting up. The individual platforms

Reviewer: ```suggestion
no longer wait for the platforms to finish setting up. The individual platforms
```

will be finished in the background. Allowing the overall startup process to

> continue and this results in a faster startup.

Reviewer: ```suggestion
continue, resulting in a faster startup.
```
